### ISSUE TITLE
spimdata-extras...

### ISSUE THREAD

**Issue by tischi** [bigdataprocessor/bigdataprocessor2#198]
@NicoKiaru 

Is this correct?

1. I do not need `spimdata-extras` anymore?
2. I now need to upload `bigdataviewer-image-loaders` to the update site?

**Comment by NicoKiaru** [comment]
Yep, that sounds correct. I can make a test once you've uploaded everything.

Best, 

Nicolas

**Comment by tischi** [comment]
@NicoKiaru 

1. What should I do with this?

<img width="445" alt="image" src="https://user-images.githubusercontent.com/2157566/192223030-44e8669b-24b2-4528-be3a-a24916ae6282.png">


2. As usual, I have trouble getting the `mark as obsolete` option to appear. Is there a new trick?

**Comment by NicoKiaru** [comment]
Ah, so you need spimdata-extras it seems in the end. So upload it.

For ome-xml, just break the dependency

**Comment by NicoKiaru** [comment]
Yep, that's correct: spimdata-extras is a dependency of image-loaders:

https://github.com/BIOP/bigdataviewer-image-loaders/blob/legacy/pom.xml

**Comment by tischi** [comment]
OK, So I guess I keep it on my update site?!

**Comment by tischi** [comment]
OK, update site is updated.
could you play around with it a bit?

**Comment by NicoKiaru** [comment]
I tested with a new Fiji and BDP update site only. It seems to work OK 👍
